presto-main/src/main/java/io/prestosql/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -411,6 +411,10 @@ private static Expression toExpression(Object value)
                         .collect(toList()));
             }
 
+            if (value instanceof Map) {
+                //TODO: How to convert map to an expression here
```

Comment:
There does not seem to exist a map construct which can directly be converted to expression, please suggest what to do here

Reply:
`toImmutableList` 🙂